filesystems/blob/src/main/scala/cromwell/filesystems/blob/BlobPathBuilderFactory.scala

```diff
@@ -33,4 +33,6 @@ final case class BlobPathBuilderFactory(globalConfig: Config, instanceConfig: Co
       new BlobPathBuilder(container, endpoint)(fsm)
     }
   }
+
+  override def priority = 100
```

Comment:
Is there somewhere that we could add documentation that we think we'll maintain? Knowing what filesystems have a priority override from default would be nice if we ever have to change this so we don't have to chase through all the filesystems.

Reply:
This is for sure a weakness of this solution. I definitely have a desire to be able to look at one place and see the relative priorities. But at the same time, that would require Cromwell core to add dependencies on ALL of the subordinate file systems, which is clearly not ideal either. I am very open to brainstorming, and since it's relatively simple it is easy to change.

Reply:
Pushed a new commit implementing the constants suggestion from mob time.